frontend/src/QuickChecklist.tsx

```diff
@@ -0,0 +1,49 @@
+import { RouteComponentProps } from '@reach/router';
+import React, { useContext } from 'react';
+import { Typography } from '@material-ui/core';
+import ReactMarkdown from 'react-markdown';
+import { getModuleDescription, getModule } from './utils/moduleHelpers';
+import { AppContext } from './context';
+import Checklists from './components/Checklists';
+
+interface ProjectProps extends RouteComponentProps {
+  moduleName?: string;
+  categoryName?: string;
+}
+
+export const QuickChecklist = (props: ProjectProps) => {
+  const { categories } = useContext(AppContext);
+
+  const categoryName = props.categoryName || "";
+  const moduleName = props.moduleName || "";
+
+  const module = getModule(categories, categoryName, moduleName);
```

Comment:
if it worth calling `getModule` if `categoryName` or `moduleName` is an empty string?